<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/tetrixIcon.png</normaloff>:/images/tetrixIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border-style:double;
	border-width:4px;
	border-color: rgb(41, 165, 19);
	color: rgb(255, 255, 255);
	font: 700 14pt &quot;Microsoft YaHei UI&quot;;
	border-radius:10px;
	padding:4px;
	min-height:40px;
}
QFrame#frameBg{	
	background-color: rgb(49, 128, 255);
}
QLabel{
	border-width:0px;
}
QLabel#labelNextPiece{
border-width:2px;
}
QFrame#frame{
	background-image: url(:/images/tetrixbg.png);
     padding:4px;
}
QPushButton{
border-width:1px;
}
QPushButton:hover{
	font: 700 16pt &quot;Microsoft YaHei UI&quot;;
}
QPushButton:pressed{
	padding-left:6px;
	padding-top:6px;
	font: 700 14pt &quot;Microsoft YaHei UI&quot;;
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frameBg">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QLCDNumber" name="lcdScore"/>
       </item>
       <item row="3" column="2">
        <widget class="QLCDNumber" name="lcdSpeed"/>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QLabel" name="labelNextPiece">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="btnExit">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>速度</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>以消除行数</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>分数</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>下一个</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="9">
        <widget class="TetrixBoard" name="frame">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>56</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLCDNumber" name="lcdRemoved"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="btnRestart">
         <property name="text">
          <string>重新开始</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TetrixBoard</class>
   <extends>QFrame</extends>
   <header location="global">tetrixboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
